service-perfplat/openapi/lpa-make-performance-platform-openapi-v1.yml

```diff
@@ -0,0 +1,366 @@
+---
+openapi: '3.0.2'
+info:
+  title: lpa-data-api-${version}
+  version: '1.0'
+servers:
+  - url: /v1
+    description: we use the path only so we can use in different contexts
+
+x-amazon-apigateway-request-validators:
+  all:
+    validateRequestParameters: true
+    validateRequestBody: true
+
+x-amazon-apigateway-gateway-responses:
```

Comment:
Can you explain why these are included in our API definition please?

Reply:
This is something that should have been removed once I moved it over entirely to docker. Slipped my mind.